code/modules/client/verbs/aooc.dm

```diff
@@ -120,6 +120,6 @@ GLOBAL_VAR_INIT(normal_aooc_colour, "#ce254f")
 			antaglisting |= M.client
 
 	for(var/client/C in antaglisting)
-		if(!C || !istype(C))
+		if(!C || !istype(C) || !is_special_character(C.mob))
```

Comment:
This prevents ghosts, dead spacemen, and admins from being able to actually tell when AOOC is toggled. Move this check to line 119 instead. And also can you add a isnewplayer check to 119 while you're at it, thanks

Reply:
fuck